src/internal/m365/collection/drive/collections_test.go

```diff
@@ -347,63 +365,63 @@ func (suite *CollectionsUnitSuite) TestPopulateDriveCollections() {
 		{
 			name: "Invalid item",
 			items: []models.DriveItemable{
-				driveRootItem(rootID),
-				driveItem(id(item), name(item), parent(drive), rootID, -1),
+				driveRootItem(),
+				driveItem(id(item), name(item), driveParentDir(drive), rootID, -1),
```

Comment:
these calls and calls to similar functions seem like incorrect usages of the helper functions by the standards that exist now (perhaps they were written when expectations were different). Could you please make another PR to update them to adhere to the new standards? It seems like most of them should be switched to just `parentDir` (or equivalent function)

Reply:
Going to keep this around in this PR.  It will get addressed in a future PR where I clean up the drive handling specifically.